ui/components/src/ui-components/table/component/Table.tsx

```diff
@@ -1156,6 +1159,23 @@ export class Table extends React.Component<TableProps, TableState> {
     return cells;
   }
 
+  private mergeClassNames(classNames1: string, classNames2: string): string{
```

Comment:
Have you checked `classnames` package? Maybe it has function to concatenate class names that you could use?

Reply:
Yep, checked it, however function provided by `classnames` package stacked classnames on top of each other even though some of the classnames were already set. I could use it, but then the final classname comes out clustered with repeated strings.

Reply:
It looks like this should work https://github.com/JedWatson/classnames#alternate-dedupe-version

Reply:
Yeep. Missed it, thanks!

Reply:
I guess it would be better to import both versions of `classnames` function (`"classnames"` and `"classnames/dedupe"`) and use dedupe version only where you used `mergeClassNames` and leave all other places to use `classnames` without dedupe as it was before.

Reply:
Makes sense. Updated